lib/features/home/home_screen/hot_bids/ui/hot_bids_widget.dart

```diff
@@ -0,0 +1,266 @@
+                    itemCount: curatedPhotos.length,
+                    separatorBuilder: (context, _) => SizedBox(width: 16),
+                    itemBuilder: (context, index) {
+                      return _hotBidsItem(
+                        curatedPhotos[index],
+                        Theme.of(context).textTheme,
+                        Theme.of(context).colorScheme,
+                        () {}, // todo open photo bottom sheets
+                      );
+                    },
+                  ),
+                ),
+              ],
+            );
+          }
+          return Container();
+        },
+      ),
+    );
+  }
+
+  Widget _hotBidsItem(
+    Photo photo,
+    TextTheme textTheme,
+    ColorScheme colorScheme,
+    VoidCallback openPhoto,
+  ) {
+    return Container(
+      width: 160,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          _hotBidPhoto(
+            photo,
+            textTheme,
+            colorScheme,
+            openPhoto,
+          ),
+          Spacer(flex: 8),
+          Text(
+            photo.photographer,
+            softWrap: true,
+            maxLines: 1,
+            overflow: TextOverflow.clip,
+            style: textTheme.titleLarge,
+          ),
+          Spacer(flex: 4),
+          Text(
+            photo.alt,
+            softWrap: true,
+            maxLines: 1,
+            overflow: TextOverflow.ellipsis,
+            style: textTheme.bodyMedium
+                ?.copyWith(color: colorScheme.onSurfaceVariant),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _hotBidPhoto(
+    Photo photo,
+    TextTheme textTheme,
+    ColorScheme colorScheme,
+    VoidCallback openPhoto,
+  ) {
+    return SizedBox(
+      height: 160,
+      child: Stack(
+        children: <Widget>[
+          ClipRRect(
+            borderRadius: BorderRadius.circular(20),
+            child: Material(
+              child: Ink.image(
+                image: NetworkImage(photo.src.medium),
+                height: 160,
+                width: 160,
+                fit: BoxFit.cover,
+                child: InkWell(
+                  onTap: openPhoto,
+                ),
+              ),
+            ),
+          ),
+          Container(
+            margin: const EdgeInsets.all(16),
+            alignment: Alignment.bottomLeft,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                color: colorScheme.background.withOpacity(0.6),
+                borderRadius: BorderRadius.circular(8),
+              ),
+              child: Padding(
+                padding: const EdgeInsets.symmetric(vertical: 4, horizontal: 8),
+                child: Text(
+                  photo.avgColor,
+                  style: textTheme.labelMedium?.copyWith(color: goldColor),
+                ),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _hotBidsHeader(
```

Comment:
This is a good candidate for a common widget since it's the same view as in live auctions and best collections widgets